js/interface.js

```diff
@@ -522,7 +522,7 @@ function hideDropZone() {
   $dropZone.removeClass('active');
 }
 
-function uploadFiles(files) {
+function uploadFiles(files) {  
```

Comment:
please set your editor to trim empty spaces, on Sublime it's:

```
"translate_tabs_to_spaces": true,
"trim_trailing_white_space_on_save": true,
```

Reply:
This is fixed now.